### PR TITLE
Add default language server for JSON

### PR DESCRIPTION
**Pull request by David-Else**

### CODE REVIEW

**Review by the-mikedavis — APPROVED** [review]

File: languages.toml [hunk 1]
```diff
@@ -121,6 +121,9 @@ scope = "source.json"
 injection-regex = "json"
 file-types = ["json"]
 roots = []
+language-server = { command = "vscode-json-language-server", args = ["--stdio"] }
```
Comment (EricCrosson) [inline languages.toml:124]: How is this normally installed? I see the [vscode-json-languageserver](https://www.npmjs.com/package/vscode-json-languageserver) but the binary is not the same name

Comment (EricCrosson) [inline languages.toml:124]: How is this normally installed? I see the [vscode-json-languageserver](https://www.npmjs.com/package/vscode-json-languageserver) but the binary is not the same name

Comment (EpsilonKu) [reply languages.toml:124]: @EricCrosson Did you tried to install it? Package name can be different than package path. I installed from AUR, it work on `vscode-json-language-server` path.

Comment (the-mikedavis) [reply languages.toml:124]: it's available under another npm package: https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#jsonls

Comment (EricCrosson) [reply languages.toml:124]: @EpsilonKu I did :slightly_smiling_face: no dice

Thanks @the-mikedavis!

**Pull request merged**